src/app/selectors/manuscript-editor.selectors.ts

```diff
@@ -28,6 +28,14 @@ export const getFocusedEditorStatePath = createSelector(getManuscriptEditorState
   get(manuscriptEditorState, 'focusedManuscriptPath')
 );
 
+export const getLastSyncTimestamp = createSelector(getManuscriptEditorState, (manuscriptEditorState) =>
```

Comment:
Not an issue now, nor may it be an issue at all. But will need to consider how we synchronise timestamps between clients. I guess the server should be the source of truth, and supply a starting timestamp when the article is first loaded? Can't rely on clients having the correct time on their computers.